app/models/manageiq/providers/redfish/physical_infra_manager/event_catcher/runner.rb

```diff
@@ -0,0 +1,38 @@
+require "server_sent_events"
+
+module ManageIQ::Providers::Redfish
+  class PhysicalInfraManager::EventCatcher::Runner \
+      < ManageIQ::Providers::BaseManager::EventCatcher::Runner
+    def monitor_events
+      event_stream.listen do |event|
+        event_monitor_running
+        @queue << event
+      end
+    end
+
+    def stop_event_monitor
+    end
+
+    def queue_event(events)
+      PhysicalInfraManager::EventParser.events_to_hashes(
```

Comment:
You probably want to fully qualify the class name to the Redfish::PhysicalInfraManager::EventParser.